### DISCUSSION TITLE
customizing the flet splash/loading screen

### DISCUSSION THREAD

**Discussion by TobiasHT5** [flet-dev/flet#1027] [Q&A] [answered]
Once a flet app is loading in the browser, it displaying a periodically growing and shrinking flet logo. 
Is there a way of replacing it with a custom logo or just removing the splash screen entirely?

**Comment by TobiasHT5** [comment, marked as answer]
I've actually gotten the answer from the docs. 
No bother..

**Comment by bobwatcherx** [comment]
why did I change it in icons/logo-animation.png but the logo didn't change in the dist folder

**Comment by bobwatcherx** [comment]
and favicon cannot change but i change the image in assets folder root

**Comment by bobwatcherx** [comment]
![Selection_003](https://user-images.githubusercontent.com/52072811/237057853-c30b4375-2845-4ff5-9efc-ae5c7079bcb0.png)

**Comment by bobwatcherx** [comment]
when I build it to web, why is the flet logo still default, I have changed the image to icons/loading-animation.png
